internal/pkg/vaultworker/password.go

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * Copyright 2018 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ * @author: Tingyu Zeng, Dell
+ * @version: 1.0.0
+ *******************************************************************************/
+
+package vaultworker
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/dghubble/sling"
+	"github.com/sethvargo/go-password/password"
+)
+
+type UserPasswd struct {
+    User string
+    Passwd string
+}
+
+func CreateCredential() (string, error) {
+	pass, err := password.Generate(8, 4, 4, false, false)
```

Comment:
perhaps it also makes sense to have these arguments configurable through the configuration.toml file, not sure

Reply:
It might be better to provide interface with default password generator and leave options open. Will address this in #77

Reply:
If you want to merge this PR ASAP, I would be okay with getting rid of the usage of this library and instead just hard-coding the function to return something like 
```go
func CreateCredential() (string, error) {
    return "TODO-implement-password-generation"
```

as this will make it clear that this needs to be refactored and will allow a proper vetting to take place and will not give the impression to anyone who reviews the code that the usage of this library was vetted when it was not

Reply:
Use this log warning as well as hard-coded value for future improvement.
lc.Warn("todo: need to implment password generation mechanism.")	
return "not-trusted-password-method", nil